typings/core.d.ts

```diff
@@ -3,7 +3,7 @@ import { SwalOptions } from './modules/options';
 export declare type SwalParams = (string | Partial<SwalOptions>)[];
 export interface SweetAlert {
     (...params: SwalParams): Promise<any>;
-    close?(namespace?: string): void;
+    close?(namespace: string): void;
```

Comment:
Was this necessary? Because it was recently merged in #802

Reply:
actually is not, it was generated by the transpiler.